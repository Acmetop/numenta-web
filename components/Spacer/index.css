/**
 * Numenta shared web components
 * MIT License (see LICENSE.txt)
 * Copyright © 2005—2016 Numenta <http://numenta.com>
 */

.spacer {
  composes: mh1 from 'tachyons';
  composes: silver from 'tachyons';
}
