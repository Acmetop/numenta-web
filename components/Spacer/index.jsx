// Numenta shared web components
// MIT License (see LICENSE.txt)
// Copyright © 2005—2016 Numenta <http://numenta.com>

import React from 'react'

import styles from './index.css'


const Spacer = () => (
  <span className={styles.spacer}>•</span>
)

export default Spacer
