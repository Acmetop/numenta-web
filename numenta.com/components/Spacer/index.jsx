import React from 'react'

import styles from './style.css'


const Spacer = () => (
  <span className={styles.spacer}>â€¢</span>
)

export default Spacer
