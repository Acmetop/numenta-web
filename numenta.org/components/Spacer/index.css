/**
 * Numenta.org HTM Community website source code
 * MIT License (see LICENSE.txt)
 * Copyright © 2005—2016 Numenta <http://numenta.com>
 */

.spacer {
  composes: mh1 from 'tachyons';
  composes: silver from 'tachyons';
}
