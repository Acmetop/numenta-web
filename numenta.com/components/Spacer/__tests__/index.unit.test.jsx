import React from 'react'
import renderer from 'react-test-renderer'

import Spacer from '../../Spacer'


describe('<Spacer /> React component unit test suite', () => {

  it('Renders correctly', () => {
    const component = renderer.create(
      <Spacer />
    )
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })

})
