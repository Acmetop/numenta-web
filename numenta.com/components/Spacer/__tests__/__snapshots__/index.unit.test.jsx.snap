exports[`<Spacer /> React component unit test suite Renders correctly 1`] = `
<span
  className="spacer">
  â€¢
</span>
`;
